stdlib/private/SwiftPrivateLibcExtras/Subprocess.swift

```diff
@@ -264,9 +264,19 @@ public enum ProcessTerminationStatus : CustomStringConvertible {
 
 public func posixWaitpid(_ pid: pid_t) -> ProcessTerminationStatus {
   var status: CInt = 0
+#if CYGWIN
+  withUnsafeMutablePointer(&status) {
```

Comment:
Are you sure you don't need the `to:` label?  We recently added it to this function: https://github.com/apple/swift/blob/master/stdlib/public/core/LifetimeManager.swift#L62